spec/factories.rb

```diff
@@ -20,8 +21,11 @@
     end
   end
 
+  factory :step_by_step_page_with_steps_but_no_draft, parent: :step_by_step_page_with_steps do
+    draft_updated_at { nil }
+  end
+
   factory :published_step_by_step_page, parent: :step_by_step_page_with_steps do
-    draft_updated_at { 3.hours.ago }
```

Comment:
This should be left in. When `mark_as_published` is called on a step-by-step, the `draft_updated_at` is set to the same date and time as `published_at`.